Apps/Examples/Examples/All Examples/CustomLocationProviderExample.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+import MapboxMaps
+
+@objc(CustomLocationProviderExample)
+internal class CustomLocationProviderExample: UIViewController, ExampleProtocol {
+
+    var mapView: MapView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let centerCoordinate = CLLocationCoordinate2D(latitude: 40.7131854, longitude: -74.0165265)
+        let options = MapInitOptions(cameraOptions: CameraOptions(center: centerCoordinate, zoom: 10))
+        mapView = MapView(frame: view.frame, mapInitOptions: options)
```

Comment:
```suggestion
        mapView = MapView(frame: view.bounds, mapInitOptions: options)
```

`frame` and `bounds` _can_ be the same, but they often are not. `frame` is a rectangle in the coordinate space of a view's superview. `bounds` is a rectangle is the view's own coordinate space. In this case, `mapView` will be a subview of `view`, and we want it to completely fill `view`, so we should set `mapView`'s frame equal to `view.bounds`.